the-blue-alliance-ios/View Controllers/Events/EventViewController.swift

```diff
@@ -58,11 +61,12 @@ class EventViewController: ContainerViewController {
             matchesViewController.matchSelected = { match in
                 self.performSegue(withIdentifier: "MatchSegue", sender: match)
             }
-        } else if segue.identifier == "EventMatchesEmbed" {
```

Comment:
for whatever reason it existed twice it looks like, I was just replacing the duplicate with the rankings segue.